optuna/_hypervolume/hssp.py

```diff
@@ -44,10 +44,12 @@ def _solve_hssp_on_unique_loss_vals(
     subset_size: int,
     reference_point: np.ndarray,
 ) -> np.ndarray:
-    assert not np.any(reference_point - rank_i_loss_vals <= 0)
+    with np.errstate(invalid="ignore"):
+        distance_from_reference_point = reference_point - rank_i_loss_vals
+    assert not np.any(distance_from_reference_point <= 0)
```

Comment:
I found out that `BaseHypervolume` actually tests `assert not np.any(diff_of_loss_vals_and_ref_point <= 0)`, so we do not need it.
```suggestion
    with np.errstate(invalid="ignore"):
        diff_of_loss_vals_and_ref_point = reference_point - rank_i_loss_vals
```